syncer/db.go

```diff
@@ -16,16 +16,14 @@ package syncer
 import (
 	"database/sql"
 	"fmt"
-	"strings"
-	"time"
-
-	"github.com/pingcap/errors"
-	"github.com/siddontang/go-mysql/mysql"
-	"go.uber.org/zap"
-
 	"github.com/pingcap/dm/dm/config"
 	tcontext "github.com/pingcap/dm/pkg/context"
 	"github.com/pingcap/dm/pkg/log"
+	"github.com/pingcap/dm/pkg/utils"
```

Comment:
We divide the import package path into three sections
* built-in packages
* dm packages
* other packags

Reply:
Done